qbft/spectest/tests/messages/commit_data_invalid.go

```diff
@@ -9,15 +9,10 @@ import (
 
 // CommitDataInvalid tests commit data len == 0
 func CommitDataInvalid() *tests.MsgSpecTest {
-	d := &qbft.CommitData{}
-	byts, _ := d.Encode()
-	msg := testingutils.SignQBFTMsg(testingutils.Testing4SharesSet().Shares[1], types.OperatorID(1), &qbft.Message{
-		MsgType:    qbft.CommitMsgType,
-		Height:     qbft.FirstHeight,
-		Round:      10,
-		Identifier: []byte{1, 2, 3, 4},
-		Data:       byts,
-	})
+	keySet := testingutils.Testing4SharesSet()
```

Comment:
I think this tests is irrelevant since we used to use dedicated proposal/ prepare/ commit/ round change data structs and each had it's own validation.

To test a message.validate() == nill we can test specifically for the use cases there and not for the type since we don't have specific types anymore